docs/_sass/minima/_base.scss

```diff
@@ -271,11 +271,24 @@ table {
   }
 }
 
-/**
- * Prevents page break to cut through content when converting to pdf
- */
 @media print {
+  /**
+  * Prevents page break to cut through content when converting to pdf
+  */
   body {
     display: block;
   }
+  /**
+  * Replaces default site header (with navigation) with a generic header name (AB-3) when converting to pdf.
```

Comment:
-> `Replaces the top navigation menu with the project name when converting to PDF`